schemas/chemotherapy.json

```diff
@@ -104,6 +104,50 @@
         "core": true,
         "displayName": "Cumulative Drug Dosage"
       }
+    },
+    {
+      "name": "dose_intensity_reduction",
+      "description": "Indicate if there was a significant reduction in dose intensity.",
+      "valueType": "string",
+      "restrictions": {
+        "codeList": "#/list/yes_no"
+      },
+      "meta": {
+        "displayName": "Dose Intensity Reduction"
+      }
+    },
+    {
+      "name": "dose_intensity_reduction_event",
```

Comment:
@hknahal is this the part of the code that comes through on the docs website?
If so, just wondering if you describe in the Notes/Scripts that `dose_intensity_reduction_event` and `dose_intensity_reduction_amount` should be submitted if `dose_intensity_reduction` is Yes?

Reply:
Thanks @kcullion Good point - I should a note to indicate this.

Reply:
@kcullion I added a "notes" section to both these fields to indicate this.